substrate/frame/revive/fixtures/contracts/HostEvmOnly.sol

```diff
@@ -7,6 +7,11 @@ contract HostEvmOnly {
             selfdestruct(recipient)
         }
     }
+    function fallback(address payable recipient) public {
```

Comment:
Don't do this. Either make it the fallback function or choose a function name which doesn't shadow the fallback function.